cluster/gce/gci/configure-helper.sh

```diff
@@ -2781,7 +2790,14 @@ function main() {
       create-kubeproxy-user-kubeconfig
     fi
     if [[ "${ENABLE_NODE_PROBLEM_DETECTOR:-}" == "standalone" ]]; then
-      create-node-problem-detector-kubeconfig ${KUBERNETES_MASTER_NAME}
+      if [[ -n "${NODE_PROBLEM_DETECTOR_TOKEN:-}" ]]; then
```

Comment:
I'm pro just removing NODE_PROBLEM_DETECTOR_TOKEN in one swoosh and not supporting both.

Reply:
IIUC that'd break this script on plain GCE